mobly/controllers/monsoon.py

```diff
@@ -105,30 +104,30 @@ def __init__(self, device=None, serialno=None, wait=1):
                 try:  # use a lockfile to ensure exclusive access
                     fcntl.lockf(self._tempfile, fcntl.LOCK_EX | fcntl.LOCK_NB)
                 except IOError as e:
-                    logging.error("device %s is in use", dev)
+                    logging.error('device %s is in use', dev)
                     continue
 
                 try:  # try to open the device
-                    self.ser = serial.Serial("/dev/%s" % dev, timeout=1)
+                    self.ser = serial.Serial('/dev/%s' % dev, timeout=1)
                     self.StopDataCollection()  # just in case
                     self._FlushInput()  # discard stale input
                     status = self.GetStatus()
                 except Exception as e:
-                    logging.exception("Error opening device %s: %s", dev, e)
+                    logging.exception('Error opening device %s: %s', dev, e)
                     continue
 
                 if not status:
-                    logging.error("no response from device %s", dev)
-                elif serialno and status["serialNumber"] != serialno:
-                    logging.error("Another device serial #%d seen on %s",
-                                  status["serialNumber"], dev)
+                    logging.error('no response from device %s', dev)
+                elif serialno and status['serialNumber'] != serialno:
```

Comment:
a bit of both. "serialNumber" comes from the monsoon itself, right? Is there a reason "serialno" is used instead of either "serial" (matching android_device) or "serialNumber" (matching the monsoon)?